doc/nmstate.service.8.in

```diff
@@ -9,20 +9,43 @@ nmstate\&.service
 nmstate\&.service invokes \fBnmstatectl service\fR command which
 apply all network state files ending with \fB.yml\fR in
 \fB/etc/nmstate\fR folder.
+
+.PP
 By default, the network state file will be renamed with the suffix
 \fB.applied\fR after applying it. This is to prevent it being applied again
 when the service runs next.
 
 With \fB/etc/nmstate/nmstate.conf\fR holding below content:
 
+.EX
 \fB
-[service]
+[service]\n
 keep_state_file_after_apply = true
 \fR
+.EE
 
 The nmstate.service will not remove network state file, just copy applied
 network stata to file with the suffix \fB.applied\fR after applying it.
 
+.PP
+By default, nmstate merge network state file with current network state,
+if you want to override interface settings without merging, please set
+`override_iface = true` in `[service]` section in
```

Comment:
It is not clear what happens if override is used, please add a bried explanation like "default values will be used".